dcos/dcos-ui
# DCOS-51098: Simplify Webpack- and Jest-setup

This aims to simplify the configuration of webpack, babel and jest.

Therefore we:

* dry down the webpack config.
* remove our custom jest-preprocessor altogether, just relying on `ts-jest` and `js-jest`. 🎉 
* remove the jison file along with its loader.
* move module aliasing into babel instead of doing it in jest *and* webpack.
* use `glob` instead of `fs` to determine what test-files to run.
* remove some seemingly obsolete test-helpers in the test-env (`requestAnimationFrame`).
* remove some cruft (`.es6`-resolving) and do some cosmetics to old code.

## Testing

* i suggest reviewing commit by commit
* check if the same amount of tests is ran.
* check if the DSL-stuff that's jison-related is still working.
* anything that could break by the changes mentioned above.

⚠️ 	if the overall change seems like a lot, have a look at the separate commits as they should be factored ok-ish. if you don't think so, i'll happily factor this PR into several ones. just let me know if that would help.

Closes DCOS-51098
